### PR TITLE
feat: empty reading history

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Different text for when reading history is empty

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-921 #done

### CODE REVIEW

File: packages/shared/src/components/squads/SelectArticle.tsx [hunk 1]
```diff
@@ -39,39 +41,50 @@ export function SquadSelectArticle({
...
+          <SquadTitleColor>article</SquadTitleColor>
+        </SquadTitle>
+        <p className="py-4 text-center">
+          {!hasData
+            ? 'Your reading history is empty! Please read at least one article by clicking on a post to start sharing with your squad'
+            : 'Pick the first article that you would like to discuss with the members of your squad'}
+        </p>
+        <Modal.StepsWrapper>
+          {({ nextStep }) => (
+            <div>
```
Comment (sshanzel) [inline packages/shared/src/components/squads/SelectArticle.tsx:57]: I think we need to use the `InfiniteScrolling` component here.

**Review by sshanzel — APPROVED** [review]
Just had one comment regarding the use of `InfiniteScrolling`

File: packages/shared/src/components/modals/SquadsBetaModal.tsx [hunk 2]
```diff
@@ -22,13 +22,13 @@ function SquadsBetaModal({
         <Modal.Body>
           <SquadsBackgroundSvg className="absolute top-0 left-0 w-full rounded-t-16" />
           <h3 className="mt-56 font-bold text-center typo-large-title">
-            You get access to
+            Wow!! You get early access to
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/modals/SquadsBetaModal.tsx:25]: I'll double check with Sab, was as per design but agreed might look weird

File: packages/shared/src/components/squads/SelectArticle.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/squads/SelectArticle.tsx:57]: Good point, I'll add a seperate ticket for that.

File: packages/shared/src/components/squads/SelectArticle.tsx [hunk 3]
```diff
@@ -27,7 +29,7 @@ export function SquadSelectArticle({
     key,
     query: READING_HISTORY_QUERY,
   };
-  const { data, isInitialLoading, isLoading } =
+  const { hasData, data, isInitialLoading, isLoading } =
```
Comment (el) [inline packages/shared/src/components/squads/SelectArticle.tsx:32]: Now that I see this again, we might want to memoize it

File: packages/shared/src/components/squads/SelectArticle.tsx [hunk 4]
```diff
@@ -39,39 +41,50 @@ export function SquadSelectArticle({
...
-          </div>
+    <>
+      <Modal.Body>
+        <SquadTitle>
+          {!hasData ? 'Read an' : 'Share the first'}{' '}
+          <SquadTitleColor>article</SquadTitleColor>
+        </SquadTitle>
+        <p className="py-4 text-center">
+          {!hasData
+            ? 'Your reading history is empty! Please read at least one article by clicking on a post to start sharing with your Squad'
```
Comment (el) [inline packages/shared/src/components/squads/SelectArticle.tsx:52]: Nitpick: I think there should be a `.` at the end since the sentence ends

**Review by el — APPROVED** [review]
Small things but good to go as it is.

File: packages/shared/src/components/squads/SelectArticle.tsx (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/components/squads/SelectArticle.tsx:32]: It's already memoized in the hook, not sure if that solves your concern?

File: packages/shared/src/components/squads/SelectArticle.tsx (hunk 4 above)
Comment (rebelchris) [reply packages/shared/src/components/squads/SelectArticle.tsx:52]: Good point, went for consistency and updated with a dot.

**Comment by rebelchris** [comment]
@el Only one question open around the memo.

File: packages/shared/src/components/squads/SelectArticle.tsx (hunk 3 above)
Comment (el) [reply packages/shared/src/components/squads/SelectArticle.tsx:32]: Ohh, is it? Then we are good to go 🥳

**Pull request merged**